smokey/src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,91 @@
+
+    public static final int BACK_LEFT_MODULE_DRIVE_MOTOR = 0; // FIXME Set back left drive motor ID
+    public static final int BACK_LEFT_MODULE_STEER_MOTOR = 0; // FIXME Set back left steer motor ID
+    public static final int BACK_LEFT_MODULE_STEER_ENCODER = 0; // FIXME Set back left steer encoder ID
+    public static final double BACK_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(0.0); // FIXME Measure and set back left steer offset
+
+    public static final int BACK_RIGHT_MODULE_DRIVE_MOTOR = 0; // FIXME Set back right drive motor ID
+    public static final int BACK_RIGHT_MODULE_STEER_MOTOR = 0; // FIXME Set back right steer motor ID
+    public static final int BACK_RIGHT_MODULE_STEER_ENCODER = 0; // FIXME Set back right steer encoder ID
+    public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(0.0); // FIXME Measure and set back right steer offset
+
+    // *****************************************************************
+    // standard stuff constants - motors rotation, etc.
+    public static final double DegreesPerRevolution = 360.0;
+    // NEO maximum RPM 
+    public static final double neoMaximumRevolutionsPerMinute = 5676;
+    // this uses the halls effect sensor when plugged into the spark max
+    // see: https://www.revrobotics.com/rev-21-1650/ where it says "42 counts per rev."
+    public static final double RevNeoEncoderTicksPerRevolution = 42;
+
+    // *****************************************************************
+    // ball handler system constants
+    public static final int BallHandlerMotorCanId = 13;
+    public static final double BallHandlerMotorDefaultSpeed = 1.0;
+    public static final double BallHandlerMotorStorageDirectionMultiplier = 1.0;
+    public static final double BallHandlerMotorRetrievalDirectionMultiplier = -1.0;
+    public static final double BallHandlerMotorMinimumAbsoluteInputValue = 0.1;
+    public static final boolean BallHandlerMotorInvertedDirection = false;
+    public static final double BallHandlerPneumaticsDeployCycleTimeSeconds = 0.5;
+    public static final double BallHandlerPneumaticsRetractCycleTimeSeconds = 0.5;
+    public static final double BallHandlerIntakeBallTimeSeconds = 2.0;
+    public static final double BallHandlerLayupBallTimeSeconds = 1.0;
+
+    public static final PneumaticsModuleType BallHandlerPneumaticsControlModuleType = PneumaticsModuleType.CTREPCM;
+    public static final int PneumaticsControlModuleNumber = 0;
+    public static final int PneumaticsControlModuleForwardChannel = 0;
+    public static final int PneumaticsControlModuleReverseChannel = 1;
+
+    // *****************************************************************
+    // input device constants
+    public static final int portDriverController = 1;
+    public static final int portCoDriverController = 2;
+
+    // *****************************************************************
+    // telescoping arm system constants
```

Comment:
should we remove the telescoping arm constants?  Or did you mean to keep them for the arm that will go on smokey?

Reply:
agree ... done